microovn/node/node.go

```diff
@@ -216,3 +221,89 @@ func ServiceWarnings(ctx context.Context, s state.State) (types.WarningSet, erro
 	}
 	return output, nil
 }
+
+// StartCentral safely starts the central service and its child service while also generating certificates
```

Comment:
This function is doing more than starting the `central` service, which may come to a surprise to someone making use of the function in the future (including future you).

Would it make more sense to call it `JoinCentral` or something like that?

Reply:
This appears to be addressed in commit "Better (enabling|disabling) of central services" later in this series.